### ISSUE TITLE
mem_advice error

### ISSUE THREAD

**Issue by ashwith2427** [AdaptiveCpp/AdaptiveCpp#1572]
While using mem_advice in my code it is giving hipSYCL::sycl::exception that mem_advice is not yet supported. I have downloaded latest hipsycl.

**Comment by illuhad** [comment]
Right, AdaptiveCpp does not have the `mem_advise` as an asynchronous enqueued operation, but it does support as an extension a synchronous `mem_advise`, which is not enqueued. This was done because I believe that most usage patterns will not actually want to enqueue the `mem_advise`.

https://github.com/AdaptiveCpp/AdaptiveCpp/blob/develop/doc/extensions.md#acpp_ext_synchronous_mem_advise

**Comment by ashwith2427** [comment]
Okay 👍
